src/test/java/com/powsybl/openloadflow/network/impl/LfNetworkLoaderImplTest.java

```diff
@@ -124,13 +124,6 @@ void networkWithControlAreasTest() {
         List<LfNetwork> lfNetworks = Networks.load(network, parameters);
         assertEquals(1, lfNetworks.size());
         LfNetwork mainNetwork = lfNetworks.get(0);
-        assertFalse(mainNetwork.hasArea());
-
-        parameters.setAreaInterchangeControl(true);
-
-        lfNetworks = Networks.load(network, parameters);
-        assertEquals(1, lfNetworks.size());
-        mainNetwork = lfNetworks.get(0);
```

Comment:
add a note in the PR description for this change (LfArea-s always created by LfNetworkLoaderImpl, even if AIC disabled)